src/test/kotlin/lotto/domain/RankTest.kt

```diff
@@ -0,0 +1,38 @@
+package lotto.domain
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+
+class RankTest : FunSpec({
+    context("로또 등수를 확인한다.") {
+        context("1등") {
+            val rank = Rank.of(6, false)
+            rank shouldBe Rank.FIRST
+        }
+
+        context("2등") {
+            val rank = Rank.of(5, true)
+            rank shouldBe Rank.SECOND
+        }
+
+        context("3등") {
+            val rank = Rank.of(5, false)
+            rank shouldBe Rank.THIRD
+        }
+
+        context("4등") {
+            val rank = Rank.of(4, false)
+            rank shouldBe Rank.FOURTH
+        }
+
+        context("5등") {
+            val rank = Rank.of(3, false)
+            rank shouldBe Rank.FIFTH
+        }
+
+        context("꽝") {
+            val rank = Rank.of(2, false)
+            rank shouldBe Rank.MISS
+        }
```

Comment:
아래와 같이 [Kotest를 활용한 Parameterized Test](https://kotlintesting.com/kotest-parameterized/)로 대체해보면 어떨까요? 😃

```suggestion
        listOf(
            row(6, false, Rank.FIRST),
            row(5, true, Rank.SECOND),
            row(5, false, Rank.THIRD),
            row(4, false, Rank.FOURTH),
            row(3, false, Rank.FIFTH),
            row(2, false, Rank.MISS),
            row(1, false, Rank.MISS),
            row(0, false, Rank.MISS),
        ).forEach { (countOfMatch, matchBonus, expected) ->
            context("countOfMatch: ${countOfMatch}, matchBonus: ${matchBonus}이면 ${expected} 이다.") {
                val actual = Rank.of(countOfMatch, matchBonus)

                actual shouldBe expected
            }
        }
```

Reply:
withData를 사용하면 intellij에서 각 테스트를 나눠서 보여줘서 withData 방식으로 적용해 보겠습니다 :)
<img width="261" alt="image" src="https://github.com/next-step/kotlin-lotto/assets/1219373/e8288e9a-158d-430a-84c5-a3d4d9f59c55">